### PR TITLE
docs: Fix local test setup scripts

### PR DESCRIPTION
**Pull request by c-pius**
**Description**

Changes proposed in this pull request:

- some minor fixes on the scripts for building KLM and deploying module templates

**Related issue(s)**

### CODE REVIEW

File: docs/developer-tutorials/local-test-setup.md [hunk 1]
```diff
@@ -71,12 +71,15 @@ This setup is deployed with the following security features enabled:
...
       ```shell
-      make docker-build
-      make docker-push
-      make local-deploy-with-watcher IMG=<image-name>:<image-tag>
+      # build your local image
+      make docker-build IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # push it to the local registry
+      make docker-push IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # deploy using the image (note the change to port 5000 which is exposed in the cluster)
+      make local-deploy-with-watcher IMG=k3d-registry.localhost:5000/<your-image-name>:<your-image-tag>
```
Comment (c-pius) [inline docs/developer-tutorials/local-test-setup.md:82]: comment: I think we need to provide the registry and especially the port change for it to work

File: docs/developer-tutorials/local-test-setup.md [hunk 2]
```diff
@@ -88,8 +91,14 @@ This setup is deployed with the following security features enabled:
...
-   kyma alpha create module -p ../template-operator --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
+   cd <local path to template operator repository>
+   
+   # generates the manifests and saves them to a template-operator.yaml file
+   make build-manifests
+   
+   # create the a ModuleTemplate CR and saves it to a template-yaml file
+   kyma alpha create module -p ./ --version 1.2.3 \
+   --registry k3d-registry.localhost:5111 --insecure --module-config-file ./module-config.yaml
```
Comment (c-pius) [inline docs/developer-tutorials/local-test-setup.md:101]: comment: at least on my machine I cannot just create the module specifying a path to the template operator repo. Therefore I changed this to cd into the needed directory.

E.g. (`../template-operator` is my local way to the repo): 

```sh
kyma alpha create module -p ../template-operator --version 1.2.3 \
   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
CAUTION: You're using an outdated version of the Kyma CLI (2.20.1). The latest released version is: 2.20.3

WARNING: This command is experimental and might change in its final version. Use at your own risk.
- Module built (8ms)
- Default CR validation succeeded (9.595s)
- component descriptor created (1ms)
X Configuring security scanning... (5ms)
Error: component.resources.0: Must validate at least one schema (anyOf);component.resources.0.version: Does not match pattern '^[v]?(0|[1-9]\d*)(?:\.(0|[1-9]\d*))?(?:\.(0|[1-9]\d*))?(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'
```

File: docs/developer-tutorials/local-test-setup.md (hunk 1 above)
Comment (Tomasz-Smelcerz-SAP) [reply docs/developer-tutorials/local-test-setup.md:82]: :+1:

**Review by Tomasz-Smelcerz-SAP — DISMISSED** [review]

File: docs/developer-tutorials/local-test-setup.md [hunk 3]
```diff
@@ -88,8 +91,14 @@ This setup is deployed with the following security features enabled:
...
 
    ```shell
-   kyma alpha create module -p ../template-operator --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
+   cd <local path to template operator repository>
+   
+   # generates the manifests and saves them to a template-operator.yaml file
+   make build-manifests
+   
+   # create the a ModuleTemplate CR and saves it to a template-yaml file
```
Comment (mmitoraj) [inline docs/developer-tutorials/local-test-setup.md:99]:
```suggestion
   # create a ModuleTemplate CR and save it to a template YAML file
```
In the previous set of comments, you used the second-person singular (user perspective), now third-person singular (command perspective), let's unify.

File: docs/developer-tutorials/local-test-setup.md [hunk 4]
```diff
@@ -71,12 +71,15 @@ This setup is deployed with the following security features enabled:
...
-      If you want to test a custom image of Lifecycle Manager, adapt the `IMG` variable in `Makefile` and run the following:
+      If you want to test a custom image of Lifecycle Manager, run the following:
 
       ```shell
-      make docker-build
-      make docker-push
-      make local-deploy-with-watcher IMG=<image-name>:<image-tag>
+      # build your local image
+      make docker-build IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # push it to the local registry
```
Comment (mmitoraj) [inline docs/developer-tutorials/local-test-setup.md:79]:
```suggestion
      # push the image to the local registry
```

File: docs/developer-tutorials/local-test-setup.md [hunk 5]
```diff
@@ -71,12 +71,15 @@ This setup is deployed with the following security features enabled:
...
+      make docker-push IMG=k3d-registry.localhost:5111/<your-image-name>:<your-image-tag>
+      # deploy using the image (note the change to port 5000 which is exposed in the cluster)
```
Comment (mmitoraj) [inline docs/developer-tutorials/local-test-setup.md:81]:
```suggestion
      # deploy Lifecycle Manager using the image (note the change to port 5000 which is exposed in the cluster)
```

File: docs/developer-tutorials/local-test-setup.md [hunk 6]
```diff
@@ -88,8 +91,14 @@ This setup is deployed with the following security features enabled:
    Adjust the path to your `template-operator` local directory or any other reference module operator accordingly.
 
    ```shell
-   kyma alpha create module -p ../template-operator --version 1.2.3 \
-   --registry k3d-registry.localhost:5111 --insecure --module-config-file ../template-operator/module-config.yaml
+   cd <local path to template operator repository>
+   
+   # generates the manifests and saves them to a template-operator.yaml file
```
Comment (mmitoraj) [inline docs/developer-tutorials/local-test-setup.md:96]:
```suggestion
   # generate the manifests and save them to a template-operator.yaml file
```
In the previous set of comments, you used the second-person singular (user perspective), now third-person singular (command perspective), let's unify.

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**